perception/traffic_light_occlusion_predictor/config/traffic_light_occlusion_predictor.param.yaml

```diff
@@ -0,0 +1,11 @@
+/**:
+  ros__parameters:
+    azimuth_occlusion_resolution: 0.15 # degree
+    elevation_occlusion_resolution: 0.08  # degree
```

Comment:
[imo] please add suffix "_deg"

Reply:
updated in commit [692e1e5](https://github.com/autowarefoundation/autoware.universe/pull/4050/commits/692e1e588e0478148fa48b140b55dcbaef4e9186)